workbench-core/example/app/src/routes/dynamicAuthorizationRoutes.ts

```diff
@@ -22,6 +22,10 @@ import {
   GetUserGroupsRequest,
   GetUserGroupsRequestParser
 } from '@aws/workbench-core-authorization/lib/models/getUserGroups';
+import {
+  GetUsersFromGroupRequest,
+  GetUsersFromGroupRequestParser
+} from '@aws/workbench-core-authorization/lib/models/getUsersFromGroup';
```

Comment:
since these are for request parsing, I would move them to the example app package. See how I did it in #730

Reply:
Fixed !